CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+## Contributing to Garble
+
+Thank you for your interest in contributing! Here are some ground rules:
+
+1. The tool's design decisions are in the [README](README.md)
+2. New features or major changes should be opened as an issue first
+3. Non-trivial contributions should be done in PRs with code review and CI
+4. Repo access can be given after someone has contributed a few times
```

Comment:
Maybe it's better not "several times", but through a vote between current contributors?

Reply:
Hmm. A voting system implies that we have some form of governance system, which is not the case as I'm the only repository owner. I don't think the project is large enough to warrant thinking about those things just yet.

I want to assume good will from active contributors for now, and see how far we can go with just that. If people start pushing to master too aggressively, or if we have other issues, I can revisit this policy.

Reply:
I'm just suggesting we avoid ambiguities in CONTRIBUTING.md.

Also good option: "access to the repository is granted only by the decision of the owner of the repository".

Reply:
Yeah you're right, this can be ambiguous or misinterpreted. I'll just remove it for now :)